packages/client/hmi-client/src/model-representation/petrinet/petrinet-service.ts

```diff
@@ -551,13 +551,13 @@ export const updateExistingModelContent = (amr: Model, amrOld: Model): Model =>
 export const cloneModelWithExtendedTypeSystem = (amr: Model) => {
 	const amrCopy = cloneDeep(amr);
 	if (amrCopy.semantics?.typing) {
-		const { name, description, schema, semantics } = amrCopy;
+		const { name, description, schema } = amrCopy.header;
 		const typeSystem = {
```

Comment:
@mwdchang do you know what this `typeSystem` is used for? Does it need to follow the structure of
```
{
  header: {...},
  model: semantics?.typing?.system
}
```

Reply:
This is @echl 's work on Catlab stratification, so he may be better to answer here if it should be addressed.

We are kind pivoting away from using Catlab stratify in favour of using MIRA stratify so this is not high priority.